### PR TITLE
Add descriptions for a bunch of ground station commands

### PR DESCRIPTION
**Pull request by tganley**

### CODE REVIEW

File: src/groundStation/system.py [hunk 1]
```diff
@@ -3233,7 +3258,7 @@ def __init__(self):
                                 'ctime': '>u4',
                             }
                         }
-                    },
+                    },UFH
```
Comment (Taylorr82) [inline src/groundStation/system.py:3261]: This looks like an error

File: src/groundStation/system.py [hunk 2]
```diff
@@ -3190,6 +3214,7 @@ def __init__(self):
                     'DFGM_GET_HK': {
                         'subPort': 3,
                         'inoutInfo': {
+                            'what': 'Fetch housekeeping data for the DFGM payload.'
```
Comment (jmmabanta) [inline src/groundStation/system.py:3217]:
```suggestion
                            'what': 'Fetch housekeeping data for the DFGM payload.',
```
Missed comma

**Review by jmmabanta — CHANGES_REQUESTED** [review]

File: src/groundStation/system.py (hunk 1 above)
Comment (Taylorr82) [reply src/groundStation/system.py:3261]: @tganley did you see this?

Comment (tganley) [reply src/groundStation/system.py:3261]: Yes thanks

**Review by jmmabanta — APPROVED** [review]

**Pull request merged**